documents/model.go

```diff
@@ -118,7 +118,7 @@ type Model interface {
 	IsDIDCollaborator(did identity.DID) (bool, error)
 
 	// AddAttribute adds a custom attribute to the model with the given value. If an attribute with the given name already exists, it's updated.
```

Comment:
Please update the comment

Reply:
I don't understand what you exactly mean.